css/newtab.css

```diff
@@ -3,6 +3,10 @@ body {
 	font-family: "Trebuchet MS", Helvetica, sans-serif;
 	font-size: 0.8em;
 	margin: 0;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-size: 100% auto;
```

Comment:
`background-size: cover` will be better. If you change size of a browser window `background-size: 100% auto;` fit image by width only.